app/src/main/res/layout/nutriment_item_list.xml

```diff
@@ -26,7 +26,7 @@
         android:layout_marginLeft="@dimen/activity_horizontal_margin"
         android:layout_marginRight="@dimen/activity_horizontal_margin"
         android:gravity="end"
-        tools:text="496 Kcal"
+        tools:text="@string/kcal_496"
```

Comment:
Probably is better to use a more generic name? I also think the Top/Bottom margins (for example) can be also referenced instead or being hardcoded

Reply:
sure